app/models/restaurant.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
If you decided to use that, you need to use that in all your files.
Remove the rule you ignored in `rubocop` and fix the rest of the files.

Reply:
this is a leftover from the test I've done enabling the magic comment cop... 
removing the comment